app/assets/stylesheets/govuk-component/_previous-and-next-navigation.scss

```diff
@@ -1,81 +1,83 @@
-.govuk-previous-and-next-navigation {
+.pub-c-pagination {
   display: block;
   margin-top: $gutter;
   margin-bottom: $gutter;
   margin-left: -$gutter-half;
   margin-right: -$gutter-half;
```

Comment:
@nickcolley I'd argue the margin scenario is a bit different because it's always the same, i.e: `margin: top right bottom left`, whereas with an options object there's no "standard' parameters to be passing, so long hand is more important then? But I guess it's personal preference 🙂